api/src/main/java/io/strimzi/api/kafka/model/JmxTransSpec.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.JmxTransOutputDefinitionTemplate;
+import io.strimzi.api.kafka.model.template.JmxTransQueryTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Representation for options to be passed into setting up the JmxTrans.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({"outputDefinitions", "queries", "resources"})
+@EqualsAndHashCode
+public class JmxTransSpec implements UnknownPropertyPreserving, Serializable {
+    public static final int DEFAULT_HEALTHCHECK_DELAY = 15;
+    public static final int DEFAULT_HEALTHCHECK_TIMEOUT = 5;
+
+    private static final long serialVersionUID = 1L;
+    protected String image;
+    private List<JmxTransOutputDefinitionTemplate> outputDefinitions = null;
+    private List<JmxTransQueryTemplate> kafkaQueries = null;
+
+    private ResourceRequirements resources;
+
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("The image to use for the JmxTrans")
+    public String getImage() {
+        return image;
+    }
+
+    public void setImage(String image) {
+        this.image = image;
+    }
+
+    @JsonProperty(required = true)
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    @Description("Defines the output hosts that will be referenced later on. " +
+            "For more information on these properties see, xref:type-JmxTransSpec-reference[`JmxTransSpec` schema reference].")
```

Comment:
Isn't that a link to basically the same place in the documentation that the `@Description` itself generates?

Reply:
@PaulRMellor can you explain this?

Reply:
I was suggesting to have references  _to_ the appendix from the related main content.

Reply:
@julianGoh17 - we don't need these links in the appendix tables. It would be useful, for example, to link down to the appendix from the related procedure in the main content